technical-fundamentals/coding/codeReview/index.mjs

```diff
@@ -1,50 +1,50 @@
 // non-declarative naming
-function mapArray(array) {
-  return array.map((el) => !!el);
+function mapToBooleans(array) {
+  return array.map(Boolean);
 }
 
 class Player {}
 let player;
 
 // declarative naming mismatch
-function getPlayer() {
+function getOrCreatePlayer() {
   if (!player) {
     player = new Player();
   }
   return player;
 }
```

Comment:
There is a Design Pattern ([Singleton example from Refactoring Guru](https://refactoring.guru/design-patterns/singleton/typescript/example#example-0)) that solves the same problem. We could implement it here.

Reply:
Singleton is generally a deprecated pattern. Global variables have been considered harmful over time.